oneview_redfish_toolkit/api/errors.py

```diff
@@ -20,7 +20,8 @@
 
 AUTH_ONEVIEW_ERRORS = ['AUTHN_AUTH_FAIL',
                        'AUTHN_AUTH_FAIL_LOGINDOMAINNOTFOUND',
-                       'AUTHORIZATION']
+                       'AUTHORIZATION',
+                       'Session.INVALID']
```

Comment:
We don't have an OneView documentation for all 401 OneView errors right?

Reply:
I dont know. =)
I found these errors by manual tests.